topology/src/test/java/io/camunda/zeebe/topology/ClusterTopologyAssert.java

```diff
@@ -86,4 +93,65 @@ public ClusterTopologyAssert hasVersion(final long version) {
     assertThat(actual.version()).isEqualTo(version);
     return this;
   }
+
+  /**
+   * Asserts that the actual topology has the same topology as the expected topology ignoring
+   * timestamps and versions.
+   */
+  public ClusterTopologyAssert hasSameTopologyAs(final ClusterTopology expected) {
+    assertThat(actual.members().keySet())
+        .containsExactlyInAnyOrderElementsOf(expected.members().keySet());
+
+    // Compare MemberStates without timestamp and version
```

Comment:
:wrench: I think this is easier to implement with something like
`assertThat(actual).usingRecursiveComparison().ignoringFields("timestamp", "version").isEqualTo(expected)`